superset-frontend/plugins/plugin-chart-echarts/src/components/Echart.tsx

```diff
@@ -27,6 +27,10 @@ import {
   Ref,
 } from 'react';
 
+import { useSelector } from 'react-redux';
+import { ExplorePageState } from 'src/explore/types';
+import * as echarts from 'echarts/core';
```

Comment:
Please avoid the full module import.

```suggestion
import { registerLocale } from 'echarts/core';
```

Reply:
Nice catch. I'll see about adding a lint rule for this.

Reply:
I made the change as suggested

Reply:
Thanks for update. LGTM